[BUG] server/utils/csvSanitizer.ts is empty and fails TypeScript module checks

Describe the bug
The project contains an empty utility file 
server/utils/csvSanitizer.ts
 (0 bytes). Because it does not export any values, it fails TypeScript compiler evaluations when imported by controllers/routes expecting sanitizer modules.

Steps to reproduce
Run npx tsc --noEmit.
Observe compiler error: server/routes.ts: error TS2306: File '.../server/utils/csvSanitizer.ts' is not a module.
Proposed Solution
Either:

Implement a basic utility export inside 
server/utils/csvSanitizer.ts
 to handle CSV cell escaping and format checks, or
Delete the file and clean up any imports targeting it if sanitization is handled elsewhere.

--- a/server/utils/csvExport.test.ts
+++ b/server/utils/csvExport.test.ts
@@ -1,0 +1,32 @@
+import { describe, expect, it } from "vitest";
+import { assessmentsToCsv } from "./csvExport";
+import { escapeCsvCell, sanitizeCsvCell } from "./csvSanitizer";
+
+describe("csvSanitizer", () => {
+  it("escapes commas, quotes, and newlines", () => {
+    expect(escapeCsvCell('Doe, "Jane"\nPatient')).toBe('"Doe, ""Jane""\nPatient"');
+  });
+
+  it("prefixes spreadsheet formula values", () => {
+    expect(sanitizeCsvCell("=HYPERLINK(\"https://example.com\")")).toBe(
+      "'=HYPERLINK(\"https://example.com\")"
+    );
+    expect(sanitizeCsvCell("  +SUM(A1:A2)")).toBe("'  +SUM(A1:A2)");
+  });
+});
+
+describe("assessmentsToCsv", () => {
+  it("exports sanitized CSV rows", () => {
+    const csv = assessmentsToCsv([
+      {
+        patientName: "Jane, Doe",
+        riskCategory: "=HIGH",
+        notes: 'Needs "follow-up"',
+      },
+    ]);
+
+    expect(csv).toBe(
+      'patientName,riskCategory,notes\\n"Jane, Doe",\'=HIGH,"Needs ""follow-up"""'
+    );
+  });
+});

--- a/server/utils/csvExport.ts
+++ b/server/utils/csvExport.ts
@@ -1,6 +1,10 @@
+import { escapeCsvCell } from "./csvSanitizer";
+
 export function assessmentsToCsv(data: Record<string, unknown>[]): string {
   if (data.length === 0) return "";
   const headers = Object.keys(data[0]);
-  const rows = data.map(row => headers.map(h => JSON.stringify(row[h] ?? "")).join(","));
-  return [headers.join(","), ...rows].join("\\n");
+  const rows = data.map((row) =>
+    headers.map((h) => escapeCsvCell(row[h])).join(",")
+  );
+  return [headers.map(escapeCsvCell).join(","), ...rows].join("\\n");
 }

--- a/server/utils/csvSanitizer.ts
+++ b/server/utils/csvSanitizer.ts
@@ -1,0 +1,25 @@
+const FORMULA_PREFIX_PATTERN = /^[=+\-@]/;
+
+export function sanitizeCsvCell(value: unknown): string {
+  if (value === null || value === undefined) {
+    return "";
+  }
+
+  let text = value instanceof Date ? value.toISOString() : String(value);
+
+  if (FORMULA_PREFIX_PATTERN.test(text.trimStart())) {
+    text = `'${text}`;
+  }
+
+  return text;
+}
+
+export function escapeCsvCell(value: unknown): string {
+  const sanitized = sanitizeCsvCell(value);
+
+  if (/[",\r\n]/.test(sanitized)) {
+    return `"${sanitized.replace(/"/g, '""')}"`;
+  }
+
+  return sanitized;
+}

